package.json

```diff
@@ -105,7 +105,6 @@
     "karma": "~1.6.0",
     "karma-chrome-launcher": "~2.0.0",
     "karma-coverage": "~1.1.1",
-    "karma-firefox-launcher": "~1.0.0",
```

Comment:
We probably want to remove `karma-phantomjs-launcher` (a few lines below) as well, right?